tests/torch_api/test_decentralized.py

```diff
@@ -78,6 +78,9 @@ def run_model(
     _init_bagua_env(rank, env)
     group = bagua.communication.new_group(ranks=list(range(nranks)))
 
+    if _rank_not_in_group(group):
+        return
+
```

Comment:
Have we changed the original semantics of `with_bagua` ?

Reply:
Yes, _rank_not_in_group is fallback, it will be re-added in the later version.